### ISSUE TITLE
WritingTool works in Calc but not Writer

### ISSUE THREAD

**Issue by chuck292000** [writingtool-org/writingtool#22]
Hello!

Environment:
I recently installed WritingTool v1.0 and also tried WritingTool (nightlies) v1.1
I am using LibreOffice v24.8.3.2 64 bit with a 64 bit Windows.

Issue:
WritingTool is not working in Writer but works fine in Calc. In Writer, when selecting Tools-->WritingTools-->Options, nothing happens, no pop-up window.

What was tried to resolve:
I have un-installed all versions of Java and reinstalled Java 17 64 bit. 
I ensured that Writer is pointing to Java install.

How can this be resolved please?
Thank you!

**Comment by eddiekls** [comment]
Exactly the same here, works in Calc but not in Writer. Tested in Linux (Tumbleweed) and MacOS Sequoia

**Comment by eddiekls** [comment]
I have just realised, it also works in Impress, just not in Writer and Draw.

**Comment by gnc-07** [comment]
Same here, I installed it for the Writer specifically but I do not seem to be able to get it to work. If anyone, or the dev, could help that would be great!

**Comment by FredKruse** [comment]
That's really strange. I can't reproduce the problem. Both work for me. Could you check whether the WritingTool option appears and is activated in LO under Tools/Options/Languages ​​and Locales/Writing Aids/Available Language Modules?

**Comment by eddiekls** [comment]
Thanks for the reply. Yes, the Writing Tool option appears and is
activated. It works well in Calc and Impress, but not at all in
Writer. 



Am Montag, dem 02.12.2024 um 01:51 -0800 schrieb Fred Kruse:
> That's really strange. I can't reproduce the problem. Both work for
> me. Could you check whether the WritingTool option appears and is
> activated in LO under Tools/Options/Languages ​​and Locales/Writing
> Aids/Available Language Modules?
> —
> Reply to this email directly, view it on GitHub [1], or unsubscribe
> [2].
> You are receiving this because you commented.Message ID:
> ***@***.***>

[1] view it on GitHub
    https://github.com/writingtool-org/writingtool/issues/22#issuecomment-2511055911
[2] unsubscribe

**Comment by gnc-07** [comment]
> That's really strange. I can't reproduce the problem. Both work for me. Could you check whether the WritingTool option appears and is activated in LO under Tools/Options/Languages ​​and Locales/Writing Aids/Available Language Modules?

Yes it is enabled in the places you mentioned. Just to make sure, could you perhaps send a screenshot of what it is supposed to look like when it is working in Writer?

**Comment by chuck292000** [comment]
Hello folks, I KNOW this is not the place for this question, but it is very easy if someone knows how to do this. I emailed LO support, but no answer. When typing in Writer, and you get autofill suggestion, what is the keyboard shortcut to select word?

Thank you so much!!
